chemprop/nn/predictors.py

```diff
@@ -132,7 +139,7 @@ def n_tasks(self) -> int:
         return self.output_dim // self.n_targets
 
     def forward(self, Z: Tensor) -> Tensor:
-        return self.ffn(Z)
+        return self.output_transform(self.ffn(Z))
 
     def train_step(self, Z: Tensor) -> Tensor:
         return self.ffn(Z)
```

Comment:
```suggestion
```
previously we had to have forward and train_step because MPNN.training_step called `preds = self.predictor.train_step(Z)` so that the loc and scale wouldn't be applied. Now `output_transform` takes care of this.